core/rest/src/main/java/org/trafodion/rest/ServerResource.java

```diff
@@ -256,11 +246,13 @@ private JSONArray pstack(String program) throws IOException {
 	            String line = scanner.nextLine();
 	            if(line.contains("pstack-ing")) {
 	                continue;
-	            } else if (line.contains("pstack")) {
-	                if(pstack == true) {
+				} else if (line.contains("pstack") || line.startsWith("--")) {
+					if (pstack == true && sb.length() > 0) {
```

Comment:
This change looks good to me. I gather that all the changes with the imports are just clean-ups unrelated to this change?